src/CabanaPD_Particles.hpp

```diff
@@ -396,6 +405,17 @@ class Particles<DeviceType, PMB, Dimension>
     friend class Comm<self_type, PMB>;
 
   protected:
+    template <class LocalMeshType, std::size_t Dim = dim>
+    std::enable_if_t<3 == Dim, void> dz( const LocalMeshType local_mesh,
+                                         const int zero[dim] )
+    {
+        dx[2] = local_mesh.measure( Cajita::Edge<Cajita::Dim::K>(), zero );
```

Comment:
@streeve : Here it seems we are setting dz = 0. However, shouldn't this be done for the case of Dim == 2 instead?

Reply:
This can be removed once the linked PR is merged

Reply:
Fixed